Tasks/AzureTestPlanV0/constants.ts

```diff
@@ -11,4 +11,7 @@ export const NOT_AUTOMATED = 'Not Automated'
 export const MERGE_THRESHOLD = 100;
 export const AUTOMATED_EXECUTION = "AutomatedExecutionPhase";
 export const AUTOMATED_PUBLISHING = "PublishingAutomatedResultsPhase";
-export const MANUALTESTS_PUBLISHING = "ManualTestResultsPublishingPhase";
\ No newline at end of file
+export const MANUALTESTS_PUBLISHING = "ManualTestResultsPublishingPhase";
+export const GOTESTSUM_INSTALL = 'install';
```

Comment:
this constant can be renamed to only INSTALL and can be used anywhere else as it is not specific to go or gotestsum